windows_containers/windows-containers-release-notes-5-x.adoc

```diff
@@ -13,10 +13,20 @@ Windows Container Support for Red Hat OpenShift enables running Windows compute
 
 The release notes for Red Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
 
+ifndef::openshift-origin[]
 [id="getting-support"]
 == Getting support
 
-You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
+// wording taken and modified from https://access.redhat.com/support/policy/updates/openshift#windows
+
+Windows Container Support for Red Hat OpenShift is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
+
+You must have this separate subscription to receive support for the Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal]. 
```

Comment:
is "the" necessary before the name of the product in this sentence: 

You must have this separate subscription to receive support for **the** Windows Container Support for Red Hat OpenShift.